includes/models/model.llms.lesson.php

```diff
@@ -364,6 +364,33 @@ public function get_video() {
 		return $this->get_embed( 'video' );
 	}
 
+	/**
+	 * Get the lesson prerequisite
+	 *
+	 * @return int [ID of the prerequisite post]
+	 */
+	public function get_prerequisite() {
+
+		$pre_req = false;
+
+		$course = $this->get_course();
+
+		// Check course level setting
+		if ( $course && $course->must_complete_sequentially() ){
+			// might be false so we are okay
+			$pre_req = $this->get_previous_lesson();
+		}
+
+		// Good ol' fashioned way
+		if ( ! $pre_req ){
```

Comment:
this condition seems unnecessary to me and would be more concisely written as

```php
if ( $course && $course->must_complete_sequentially() ) [
 ...
} elseif ( $this->has_prerequisite ) {
 ...
}

Reply:
I agree

Reply:
I've made a similar change to the `has_prerequisite()` method as well